components/coding/profileV2/AssignmentSectionComponent.tsx

```diff
@@ -16,6 +16,30 @@ import {
 import { useAuth } from "../../../lib/authContext";
 import ExpandableContainer from "../ExpandableContainer";
 
+export const returnWrapStyling = (
+  assignment: UserAssignmentSubmissionsData
+) => {
+  let wrapStyle = "";
+  if (assignment.coding_assignment.assignment_name.length >= 3) {
+    wrapStyle = "truncate";
+  } else {
+    null;
+  }
+  return wrapStyle;
+};
+
+export const returnParentStyling = (
+  assignment: UserAssignmentSubmissionsData
+) => {
+  let parentStyle = "";
+  if (assignment.coding_assignment.assignment_name.length >= 8) {
+    parentStyle = "w-[100px] hover:w-full";
```

Comment:
haven't seen the syntax of 100px wrapped in brackets - what does that do? i try to avoid using absolute sizes (100) as opposed to relative (1/2, 1/3, full etc.) as it can cause unintended issues with mobile responsiveness

Reply:
shouldn't have "hover:w-full" over here... when I hover over the assignment name, the name jumps to the left side of the table.

were you trying to do something different here?

Reply:
this is what it does when i hover over: 

<img width="1086" alt="Screen Shot 2022-12-19 at 2 48 18 PM" src="https://user-images.githubusercontent.com/98791180/208541580-1ff198c3-ecb3-4eab-a47d-d128ab667f40.png">

Reply:
I was attempting to show the entire title if it's long on hover -- but you're right that it doesn't make sense for it to jump left if it's short.  I think just the truncate on mobile will be sufficient.  I think it's best to remove the hover also.